backend/src/constant/options.ts

```diff
@@ -38,6 +39,12 @@ export const VALID_OPTIONS: ValidOptions = {
     min: 1,
     max: 5,
   },
+  radius: {
+    type: 'number',
+    isRequired: false,
+    min: 0,
+    max: Infinity,
```

Comment:
uhh you sure?

Reply:
sebenernya yg ga match itu dokumentasinya, udah bener min nya 0